src/data/matrix/notation.lean

```diff
@@ -78,7 +78,7 @@ section val
 
 @[simp] lemma cons_val_zero (x : α) (u : fin m → α) : vec_cons x u 0 = x := rfl
 
-@[simp] lemma cons_val_zero' (h : 0 < m.succ) (x : α) (u : fin m → α) :
+lemma cons_val_zero' (h : 0 < m.succ) (x : α) (u : fin m → α) :
```

Comment:
This looks like a fine `simp` lemma; any particular reason to remove it as such?

Reply:
Linter complained with 
```
#print matrix.cons_val_zero' /- simp can prove this:
  by simp only [fin.mk_zero_eq_zero, matrix.cons_val_zero]
One of the lemmas above could be a duplicate.
```
I can reorder the lemmas in `data.matrix.notation` to make this a simp lemma again.

Reply:
No, it's fine like this.